src/Table/TableHead.js

```diff
@@ -84,11 +85,18 @@ class TableHead extends Component {
       }
     )
 
+    const thStyle = {}
+
+    if (width !== undefined && width >= 0) {
```

Comment:
Here you can use a simpler comparison like this 
```suggestion
    if (width) {
```
This comparison will deal with all falsy values. https://developer.mozilla.org/pt-BR/docs/Glossario/Falsy

Reply:
:+1: